### DISCUSSION TITLE
password not saved in bookmarks

### DISCUSSION THREAD

**Discussion by mailinglists35** [iterate-ch/cyberduck#14530] [Q&A]
click "open connection" button. fill in the credentials and server data. server type ftp-ssl. custom server port. check "save password" check box. click "connect".

attempt to add this connection to bookmarks - it will be added but without the saved password.

expect: add the connection including the saved password.

version 8.5.9 64bit windows 10 21h2

**Comment by AliveDevil** [comment]
Bookmarks aren't shared with credentials (except username).

Credentials are saved in your operating systems native keychain/credential vault (that is _Keychain Access.app_ on MacOS, and _Windows Credential Manager_ on Windows).

If you are referring to the empty password box on creating the bookmark, that is just a display issue of the bookmark window not getting filled in immediately.
